### ISSUE TITLE
Add filters syntax coloring

### ISSUE THREAD

**Issue by gomoku** [gorhill/uBlock#2881]
I'm wondering where is a filters syntax coloring? Do anybody imagine a code editor without it? Working with thousands filters with no colored syntax? Every real code editor has a colored syntax. In uBlock everything (in 'My filters' tab or when viewing a filters list from '3rd party filters' tab directly in a new tab) is black and mixed-up, unreadable, even comments are not grayed out. As for the second one (viewing a filters list directly in a new tab) there is no colored syntax in ADP too, but in ADP at least when you view/edit filters list in 'filters settings', there is a partially colored syntax like exceptions are green, domains are blue and comments are gray, but it still lacks of coloring $xxxxx things, numbers, brackets, equal signs, etc. [I can't believe it](https://i.ytimg.com/vi/MHIKlEAxFr0/maxresdefault.jpg)

**Comment by anewuser** [comment]
> Do **NOT**:
>     Submit design ideas.
>     Submit feature requests.
>     Submit "revolutionary ideas".

https://github.com/gorhill/uBlock/blob/master/CONTRIBUTING.md#submitting-issues

**Comment by gomoku** [comment]
My apology, I've just missed out the guidelines, despite the fact that I've read the contributing guide right before I've created the issue, just precisely in order to not create unnecessarily issues in a wrong place...because I've been almost sure I saw exactly these guidelines (quoted by @anewuser) before. What an irony of fate. What can I say now?  I can just quote myself: [I can't believe it](https://i.ytimg.com/vi/MHIKlEAxFr0/maxresdefault.jpg).
